scripts/stream2mtz

```diff
@@ -1,37 +1,49 @@
 #!/usr/bin/env python
-
+"""
+Convert CrystFEL stream file to an mtz for processing in careless
+"""
 import argparse
 import reciprocalspaceship as rs
 
-doc = """\
-Convert CrystFEL stream to an mtz with geometric metadata
-"""
-
 parser = argparse.ArgumentParser(
-    doc, formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    formatter_class=argparse.RawTextHelpFormatter, description=__doc__
+)
 
 parser.add_argument(
-    'stream', help='File in CrystFEL stream format. Must and with .stream')
+    "stream", help="File in CrystFEL stream format. Must end with .stream"
+)
 parser.add_argument(
-    '-o',
-    '--out',
+    "-o",
+    "--out",
     type=str,
-    help='Output filename. If nothing specified, will use <streamname>.mtz',
-    default=None)
-parser.add_argument('-g',
-                    '--spacegroup',
-                    type=int,
-                    help='Space group number for the output mtz',
-                    default=1)
+    help="Output filename. If nothing specified, will use <streamname>.mtz",
+    default=None,
+)
+parser.add_argument(
+    "-sg",
```

Comment:
I never put multiple letters after a single dash. Is it okay if we revert this to `-g` or `-s`?

Reply:
sure -- we can go back to `-g`.